
 Network yanpitangui-dotnet-v2_backend  Creating
 Network yanpitangui-dotnet-v2_backend  Created
 Volume "yanpitangui-dotnet-v2_postgres-socket"  Creating
 Volume "yanpitangui-dotnet-v2_postgres-socket"  Created
 Volume "yanpitangui-dotnet-v2_uds-sockets"  Creating
 Volume "yanpitangui-dotnet-v2_uds-sockets"  Created
 Container yanpitangui-dotnet-v2-nats-1  Creating
 Container yanpitangui-dotnet-v2-rinha-db-1  Creating
 Container yanpitangui-dotnet-v2-nats-1  Created
 Container yanpitangui-dotnet-v2-rinha-db-1  Created
 Container yanpitangui-dotnet-v2-worker-1  Creating
 Container backend-2  Creating
 Container backend-1  Creating
 Container yanpitangui-dotnet-v2-worker-1  Created
 Container backend-1  Created
 Container backend-2  Created
 Container yanpitangui-dotnet-v2-nginx-1  Creating
 Container yanpitangui-dotnet-v2-nginx-1  Created
Attaching to backend-1, backend-2, nats-1, nginx-1, rinha-db-1, worker-1
nats-1  | [1] 2025/08/14 21:05:51.956292 [INF] Starting nats-server
nats-1  | [1] 2025/08/14 21:05:51.956343 [INF]   Version:  2.11.7
nats-1  | [1] 2025/08/14 21:05:51.956346 [INF]   Git:      [df44964]
nats-1  | [1] 2025/08/14 21:05:51.956349 [INF]   Cluster:  my_cluster
nats-1  | [1] 2025/08/14 21:05:51.956351 [INF]   Name:     NBXW42RM7K44NQ66RPZWI4V66QDTE26BAT4RXSBYTWTRXY6EYOREOJ3X
nats-1  | [1] 2025/08/14 21:05:51.956354 [INF]   ID:       NBXW42RM7K44NQ66RPZWI4V66QDTE26BAT4RXSBYTWTRXY6EYOREOJ3X
nats-1  | [1] 2025/08/14 21:05:51.956371 [INF] Using configuration file: /etc/nats/nats-server.conf (sha256:c7aaa34d64beb74e7c700fd3504b40b01deb1275f6567fb81ed3809ff03a5fe4)
nats-1  | [1] 2025/08/14 21:05:51.959825 [INF] Starting http monitor on 0.0.0.0:8222
nats-1  | [1] 2025/08/14 21:05:51.961669 [INF] Listening for client connections on 0.0.0.0:4222
nats-1  | [1] 2025/08/14 21:05:51.962025 [INF] Server is ready
nats-1  | [1] 2025/08/14 21:05:51.964374 [INF] Cluster name is my_cluster
nats-1  | [1] 2025/08/14 21:05:51.966688 [INF] Listening for route connections on 0.0.0.0:6222
rinha-db-1  | The files belonging to this database system will be owned by user "postgres".
rinha-db-1  | This user must also own the server process.
rinha-db-1  | 
rinha-db-1  | The database cluster will be initialized with locale "en_US.utf8".
rinha-db-1  | The default database encoding has accordingly been set to "UTF8".
rinha-db-1  | The default text search configuration will be set to "english".
rinha-db-1  | 
rinha-db-1  | Data page checksums are disabled.
rinha-db-1  | 
rinha-db-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
rinha-db-1  | creating subdirectories ... ok
rinha-db-1  | selecting dynamic shared memory implementation ... posix
rinha-db-1  | selecting default "max_connections" ... 100
rinha-db-1  | selecting default "shared_buffers" ... 128MB
rinha-db-1  | selecting default time zone ... UTC
rinha-db-1  | creating configuration files ... ok
rinha-db-1  | running bootstrap script ... ok
rinha-db-1  | sh: locale: not found
rinha-db-1  | 2025-08-14 21:05:58.628 UTC [42] WARNING:  no usable system locales were found
rinha-db-1  | performing post-bootstrap initialization ... ok
rinha-db-1  | syncing data to disk ... ok
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | Success. You can now start the database server using:
rinha-db-1  | 
rinha-db-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
rinha-db-1  | 
rinha-db-1  | initdb: warning: enabling "trust" authentication for local connections
rinha-db-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
rinha-db-1  | waiting for server to start....2025-08-14 21:06:05.726 UTC [72] LOG:  starting PostgreSQL 17.5 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
rinha-db-1  | 2025-08-14 21:06:05.730 UTC [72] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1  | 2025-08-14 21:06:05.741 UTC [75] LOG:  database system was shut down at 2025-08-14 21:06:04 UTC
rinha-db-1  | 2025-08-14 21:06:05.749 UTC [72] LOG:  database system is ready to accept connections
rinha-db-1  |  done
rinha-db-1  | server started
rinha-db-1  | CREATE DATABASE
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
rinha-db-1  | CREATE TABLE
rinha-db-1  | CREATE INDEX
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | waiting for server to shut down....2025-08-14 21:06:07.538 UTC [72] LOG:  received fast shutdown request
rinha-db-1  | 2025-08-14 21:06:07.541 UTC [72] LOG:  aborting any active transactions
rinha-db-1  | 2025-08-14 21:06:07.632 UTC [72] LOG:  background worker "logical replication launcher" (PID 78) exited with exit code 1
rinha-db-1  | 2025-08-14 21:06:07.727 UTC [73] LOG:  shutting down
rinha-db-1  | 2025-08-14 21:06:07.731 UTC [73] LOG:  checkpoint starting: shutdown immediate
backend-2   | backend-2
backend-1   | backend-1
rinha-db-1  | 2025-08-14 21:06:07.832 UTC [95] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.833 UTC [96] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.833 UTC [98] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.927 UTC [101] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.927 UTC [97] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.927 UTC [94] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:07.928 UTC [100] FATAL:  the database system is shutting down
nginx-1     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
rinha-db-1  | 2025-08-14 21:06:07.926 UTC [99] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.127 UTC [102] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.130 UTC [103] FATAL:  the database system is shutting down
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1     | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1     | /docker-entrypoint.sh: Configuration complete; ready for start up
rinha-db-1  | 2025-08-14 21:06:08.226 UTC [107] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.230 UTC [110] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.327 UTC [109] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.327 UTC [108] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.328 UTC [105] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.328 UTC [112] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.329 UTC [111] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.426 UTC [106] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.427 UTC [113] FATAL:  the database system is shutting down
rinha-db-1  | 2025-08-14 21:06:08.426 UTC [104] FATAL:  the database system is shutting down
backend-1   | Unhandled exception. Npgsql.PostgresException (0x80004005): 57P03: the database system is shutting down
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__235.MoveNext() + 0xc43
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x50
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1d8
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|214_1>d.MoveNext() + 0x6de
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<Open>d__214.MoveNext() + 0x873
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.PoolingDataSource.<OpenNewConnector>d__36.MoveNext() + 0x666
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.PoolingDataSource.<<Get>g__RentAsync|33_0>d.MoveNext() + 0xacf
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlConnection.<>c__DisplayClass165_0.<<StartBindingScope>g__StartBindingScopeAsync|0>d.MoveNext() + 0x35f
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlConnection.<<Open>g__PerformMultiplexingStartupCheck|42_1>d.MoveNext() + 0x3b6
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x120
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x2a8
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__9>d.MoveNext() + 0x57c
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<<Main>$>d__0.MoveNext() + 0x61e
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<Main>(String[] args) + 0x26
backend-1   |   Exception data:
backend-1   |     Severity: FATAL
backend-1   |     SqlState: 57P03
backend-1   |     MessageText: the database system is shutting down
backend-1   |     File: backend_startup.c
backend-1   |     Line: 292
backend-1   |     Routine: BackendInitialize
backend-2   | Unhandled exception. Npgsql.PostgresException (0x80004005): 57P03: the database system is shutting down
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__235.MoveNext() + 0xc43
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x50
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1d8
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|214_1>d.MoveNext() + 0x6de
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<Open>d__214.MoveNext() + 0x873
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.PoolingDataSource.<OpenNewConnector>d__36.MoveNext() + 0x666
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.PoolingDataSource.<<Get>g__RentAsync|33_0>d.MoveNext() + 0xacf
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlConnection.<>c__DisplayClass165_0.<<StartBindingScope>g__StartBindingScopeAsync|0>d.MoveNext() + 0x35f
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlConnection.<<Open>g__PerformMultiplexingStartupCheck|42_1>d.MoveNext() + 0x3b6
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x120
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x2a8
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__9>d.MoveNext() + 0x57c
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<<Main>$>d__0.MoveNext() + 0x61e
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<Main>(String[] args) + 0x26
backend-2   |   Exception data:
backend-2   |     Severity: FATAL
backend-2   |     SqlState: 57P03
backend-2   |     MessageText: the database system is shutting down
backend-2   |     File: backend_startup.c
backend-2   |     Line: 292
backend-2   |     Routine: BackendInitialize
worker-1    | PoolConfig { RouterPoolSize = 3 }
worker-1    | PipelineConfig { HandlePaymentParallelism = 20, PersistPaymentsParallelism = 10, GroupSize = 100, Timeout = 10 }
rinha-db-1  | 2025-08-14 21:06:08.759 UTC [73] LOG:  checkpoint complete: wrote 927 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.985 s, sync=0.028 s, total=1.032 s; sync files=311, longest=0.004 s, average=0.001 s; distance=4277 kB, estimate=4277 kB; lsn=0/191A6A0, redo lsn=0/191A6A0
rinha-db-1  | .2025-08-14 21:06:08.830 UTC [72] LOG:  database system is shut down
rinha-db-1  |  done
rinha-db-1  | server stopped
rinha-db-1  | 
rinha-db-1  | PostgreSQL init process complete; ready for start up.
rinha-db-1  | 
[Kbackend-1 exited with code 139
[Kbackend-2 exited with code 139
rinha-db-1  | 2025-08-14 21:06:09.127 UTC [1] LOG:  starting PostgreSQL 17.5 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
rinha-db-1  | 2025-08-14 21:06:09.127 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
rinha-db-1  | 2025-08-14 21:06:09.127 UTC [1] LOG:  listening on IPv6 address "::", port 5432
rinha-db-1  | 2025-08-14 21:06:09.133 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1  | 2025-08-14 21:06:09.141 UTC [117] LOG:  database system was shut down at 2025-08-14 21:06:08 UTC
rinha-db-1  | 2025-08-14 21:06:09.226 UTC [1] LOG:  database system is ready to accept connections
nginx-1     | 2025/08/14 21:06:10 [crit] 31#31: *1 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:10 [crit] 31#31: *1 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:15 [error] 31#31: *4 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:20 [error] 31#31: *5 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:25 [crit] 31#31: *6 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:25 [crit] 31#31: *6 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:30 [error] 31#31: *9 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:35 [error] 31#31: *10 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:41 [crit] 31#31: *11 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:41 [crit] 31#31: *11 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:46 [error] 31#31: *14 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:51 [error] 31#31: *15 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:56 [crit] 31#31: *16 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:06:56 [crit] 31#31: *16 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/14 21:07:01 [error] 31#31: *19 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
[Knginx-1 exited with code 0
[Kworker-1 exited with code 0
rinha-db-1  | 2025-08-14 21:07:06.655 UTC [1] LOG:  received fast shutdown request
nats-1      | [1] 2025/08/14 21:07:06.656400 [INF] Trapped "terminated" signal
nats-1      | [1] 2025/08/14 21:07:06.659745 [INF] Initiating Shutdown...
nats-1      | [1] 2025/08/14 21:07:06.660640 [INF] Server Exiting..
rinha-db-1  | 2025-08-14 21:07:06.659 UTC [1] LOG:  aborting any active transactions
rinha-db-1  | 2025-08-14 21:07:06.728 UTC [1] LOG:  background worker "logical replication launcher" (PID 120) exited with exit code 1
rinha-db-1  | 2025-08-14 21:07:06.728 UTC [115] LOG:  shutting down
rinha-db-1  | 2025-08-14 21:07:06.731 UTC [115] LOG:  checkpoint starting: shutdown immediate
rinha-db-1  | 2025-08-14 21:07:06.757 UTC [115] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.005 s, total=0.029 s; sync files=5, longest=0.003 s, average=0.001 s; distance=8 kB, estimate=8 kB; lsn=0/191C6E8, redo lsn=0/191C6E8
[Knats-1 exited with code 0
rinha-db-1  | 2025-08-14 21:07:06.828 UTC [1] LOG:  database system is shut down
